updates/upload.go

```diff
@@ -30,9 +30,9 @@ func handleProbationUploadMessage(update tgbotapi.Update, user users.User) (tgbo
 
 func getFile(update MediaUpdate) ([]byte, error) {
 	var fileConfig tgbotapi.FileConfig
-	if len(update.Update.Message.Photo) > 0 {
-		// FIX: will this always stay 2?
-		fileConfig.FileID = update.Update.Message.Photo[2].FileID
+	numPhotos := len(update.Update.Message.Photo)
+	if numPhotos > 0 {
+		fileConfig.FileID = update.Update.Message.Photo[numPhotos-1].FileID
```

Comment:
I encountered an issue when testing things out, sending a low-quality png photo (as a workout), and accidentally ran into this error: _panic: runtime error: index out of range [2] with length 1_

Reply:
Yes, you're totally right here.
However, this is taking the highest quality version by default, I'm wondering whether that's the best option.
It is an improvment regardless! Thank you!